ansible/playbooks/roles/opensearch/tasks/patch-log4j.yml

```diff
@@ -61,8 +61,9 @@
           _archive_root_dir: >-
             {{ unarchive_list_files.files | first | dirname }}
 
-- name: Restart opendistro-performance-analyzer service
-  systemd:
-    name: opendistro-performance-analyzer
-    state: restarted
-  when: log4j_patch.changed
+# TODO: add instllation of PerfAn to opensearch role
```

Comment:
This will be added?

Reply:
Yeap. I've implemented this feature now in ansible/playbooks/roles/opensearch/tasks/install-ops.yml